File: wehelp-week2/assign2-3.py
#建立第一組數字
def func3(index):
    base = [25, 23, 20, 21] #第0組
    group=index//4  #算第幾組
    pos=index%4     #算位置
    answer=base[pos]-group*2
    print(answer)  

func3(1)
func3(5)
func3(10)
func3(30) 

File: wehelp-week2/assign2-3.js
function func3(index) {
    let base = [25, 23, 20, 21];

    let group = Math.floor(index / 4);
    let pos = index % 4;

    let answer = base[pos] - group * 2;

    console.log(answer);
}

func3(1);
func3(5);
func3(10);
func3(30);